web/tests/compose_actions.test.cjs

```diff
@@ -581,7 +581,6 @@ test("quote_message", ({disallow, override, override_rewire}) => {
         reply_type: "personal",
     };
     override(message_lists.current, "selected_id", () => 100);
-    override(message_lists.current, "selected_message", () => selected_message);
```

Comment:
Is it easy to extend these tests to verify the new logic? If not, we don't have to.